src/test/java/jenkins/plugins/office365connector/TimeUtilsTest.java

```diff
@@ -0,0 +1,67 @@
+package jenkins.plugins.office365connector;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+
+/**
+ * @author Damian Szczepanik (damianszczepanik@github)
+ */
+public final class TimeUtilsTest {
+
+    @Test
+    public void dateToString_ReturnsDateAsString() {
+
+        // given
+        long date = 1506717341515L;
+
+        // when
+        String format = TimeUtils.dateToString(date);
+
+        // then
+        // validates only the date as the timezone and local can be different
+        assertThat(format).startsWith("2017-09-29");
```

Comment:
yes, there is 2 hours off but could be more - depends what is Locale where the test is executed - but having this 'lack of precision' means that most of users get notification according to their timezone and locale :)